server/render.js

```diff
@@ -55,25 +72,25 @@ function render(req, res, next) {
       </Provider>
     );
 
-    const respond = () => {
+    const respond = (timeout = false) => {
       const body = renderToString(app);
 
       return res.send(
         renderPage({
-          body,
-          state: store.getState(),
+          body: timeout ? body : '',
+          state: timeout ? store.getState() : {},
```

Comment:
What kind of result does this eventually return to the user?

Reply:
When the request is timed out, it returns the same as if SSR was disabled, and that is handled here:
https://github.com/webkom/lego-webapp/blob/2b24f8b9d19cfe77426d3dbd99c13bbcbb84b518/app/index.js#L29

Looks like the logic was broken in my last change :laughing: